pkg/config/config.go

```diff
@@ -47,6 +47,8 @@ type Configuration struct {
 
 	Docker   *Docker   `yaml:"docker,omitempty"`
 	Deployer *Deployer `yaml:"deployer,omitempty"`
+
+	CustomBumpStrategies map[string]string `yaml:"custom_bump_strategies,omitempty"`
```

Comment:
```suggestion
	BumpStrategies map[string]string `yaml:"bump_strategies,omitempty"`
```

Custom doesn't bring much here ?